src/controller/Start.java

```diff
@@ -1,54 +1,110 @@
 package controller;
 
-import java.awt.event.ActionEvent;
+import java.awt.Font;
+import java.awt.Color;
 import java.awt.event.ActionListener;
+import java.awt.event.ActionEvent;
+
 
+import javax.swing.SwingConstants;
 import javax.swing.JPanel;
 
 /**
  * 
  * Start.java Start ist ein Controller zur verwaltung von aktionen des start
  * screens.
  * 
- * @author Nina
+ * @author Nina, Niklas E.
```

Comment:
Das wollte ich bei mir in dem Refactor Ticket entfernen - mache ich dann bei dir auch ;) War ja bei mir auto-generiert. Ist jetzt aber egal, ich entfernte die ja sowieso :)

Reply:
Alles klar.

Reply:
ja genau :)